jboss-modules/build.xml

```diff
@@ -182,11 +182,11 @@
         </module-def>
 
         <module-def name="org.jboss.resteasy.resteasy-spring">
-            <maven-resource-bundle group="org.jboss.resteasy" artifact="resteasy-spring" subdir="bundled/resteasy-spring-jar"/>
+            <maven-resource-bundle group="org.jboss.resteasy.spring" artifact="resteasy-spring" subdir="bundled/resteasy-spring-jar"/>

```

Comment:
This way we can workaround current jboss-module solution (using the new resteasy-spring components without changing anything in jboss-module descriptors.

Provement:

```bash
$ unzip -l resteasy-jboss-modules-5.0.0-SNAPSHOT.zip | grep spring                                                                                                                                                                                                                                              21:44:28
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/main/
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/main/bundled/
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/main/bundled/resteasy-spring-jar/
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring-web/
        0  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring-web/main/
    54992  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/main/bundled/resteasy-spring-jar/resteasy-spring-1.0.0.Final.jar
     1183  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring/main/module.xml
    17952  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring-web/main/resteasy-spring-web-1.0.0.Final.jar
     1184  07-27-2021 21:43   org/jboss/resteasy/resteasy-spring-web/main/module.xml
```

```bash
$ unzip -p resteasy-jboss-modules-5.0.0-SNAPSHOT.zip org/jboss/resteasy/resteasy-spring/main/module.xml | grep spring                                                                                                                                                                                           21:46:02
<module xmlns="urn:jboss:module:1.9" name="org.jboss.resteasy.resteasy-spring">
```

Reply:
I think we just need to completely remove the modules and references to them in the main project.